script/install-openldap

```diff
@@ -110,6 +110,6 @@ chmod g+r /etc/ssl/private/ldap01_slapd_key.pem
 chmod o-r /etc/ssl/private/ldap01_slapd_key.pem
 
 # Drop packets on a secondary port used to specific timeout tests
-iptables -A OUTPUT -p tcp -j DROP --dport 8389
+iptables -A INPUT -p tcp -j DROP --dport 8389
```

Comment:
DigitalOcean [has a good write-up of iptables theory](https://www.digitalocean.com/community/tutorials/a-deep-dive-into-iptables-and-netfilter-architecture).

In Travis CI, the test suite executes on the integration host itself. The `-A OUTPUT` bit means that the packets are dropped before the kernel even tries sending them anywhere. In contrast, with Vagrant, the test suite runs on the hypervisor and connects to a "remote" integration host. The hypervisor is unlikely to be doing any of its own output filtering, so the packets are sent to the guest VM. The guest kernel doesn't have anything listening on tcp/8389, so it sends a RST before even consulting the `OUTPUT` chain.

With the switch to `INPUT`, on Travis, there's no effective change. TCP SYN's to 8389 will make it through the `OUTPUT` chain intact and actually be sent to localhost:8389. At that point, the `INPUT` chain for that port will drop the SYN rather than send a RST. That means from the unit tests' perspective, this is a noop.

With Vagrant, now that the guest is dropping tcp/8389 on `INPUT`, the guest kernel will stop sending RST's and just ignore the connection attempt. That brings it in line with what Travis is doing, and is what we need to do the test.